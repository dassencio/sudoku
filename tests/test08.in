2 5
 _  _  _  _  6  _  _  _  2  10
 _  3  _  _  _  _  _  _  _  1
 5  _ 10  7  _  _  1  6  9  _
 _  _  3  _  _  7  _  _  5  2
 _  8  _  _  _  _ 10  5  _  _
 _  _  1 10  _  _  _  _  3  _
 7  5  _  _  9  _  _  3  _  _
 _ 10  2  3  _  _  7  1  _  5
 1  _  _  _  _  _  _  _  7  _
10  4  _  _  _  8  _  _  _  _
