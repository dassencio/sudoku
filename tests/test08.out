 9  1  4  8  6  5  3  7  2 10
 2  3  7  5 10  6  4  9  8  1
 5  2 10  7  8  3  1  6  9  4
 6  9  3  4  1  7  8 10  5  2
 3  8  9  6  2  1 10  5  4  7
 4  7  1 10  5  2  9  8  3  6
 7  5  6  1  9  4  2  3 10  8
 8 10  2  3  4  9  7  1  6  5
 1  6  8  2  3 10  5  4  7  9
10  4  5  9  7  8  6  2  1  3
